src/components/molecules/help-modal/index.tsx

```diff
@@ -8,40 +8,42 @@ import whatIsLiquidStaking from "../../../assets/images/what-is-liquid-staking.p
 import whatIsDeposit from "../../../assets/images/what-is-deposit.png";
 import whatIsLending from "../../../assets/images/what-is-lending.png";
 import fees from "../../../assets/images/fees.png";
-import { Text } from "../../atoms/typography";
+import { Heading, Text } from "../../atoms/typography";
 import { Trigger } from "@radix-ui/react-alert-dialog";
-import { imageStyle, linkStyle } from "./style.css";
+import { imageStyle } from "./style.css";
 import { YieldType } from "@stakekit/api-hooks";
 import { formatCountryCode } from "../../../utils/formatters";
 import { useGeoBlock } from "../../../hooks/use-geo-block";
 import { ReactNode } from "react";
 import { SKAnchor } from "../../atoms/anchor";
-import { useRegionCodeName } from "../../../hooks/use-region-code-names";
+import { Button } from "../../atoms/button";
 
 type ModalType =
   | { type: "main" }
   | { type: "fees" }
-  | ({ type: "geoBlock" } & Exclude<ReturnType<typeof useGeoBlock>, false>)
-  | { type: YieldType };
+  | ({ type: "geoBlock" } & Exclude<ReturnType<typeof useGeoBlock>, false> & {
+        regionCodeName: string | undefined;
+      })
+  | { type: YieldType }
+  | { type: "getInTouch" }
+  | { type: "whatIsStakeKit" };
 
 type HelpModalProps = {
   modal: ModalType;
+  customTrigger?: ReactNode;
 };
 
-export const HelpModal = ({ modal }: HelpModalProps) => {
+export const HelpModal = ({ modal, customTrigger }: HelpModalProps) => {
   const { t, i18n } = useTranslation();
 
-  const regionCodeName = useRegionCodeName(
-    modal.type === "geoBlock" ? modal.regionCode : undefined
-  );
-
   const getContent = (
```

Comment:
I believe that this function can be moved outside side the component body, to increase readability.

Reply:
It actually uses translation function hook, so only way is to create this to be custom hook